proto/rig/api/v1/capsule/service.proto

```diff
@@ -438,6 +438,9 @@ message DeployOutcome {
   // The objects are both the outcome of what the platform generated for
   // the Kubernetes cluster directly and what plugins are further adding.
   repeated KubernetesObject kubernetes_objects = 3;
+  // In case generation of kubernetes files failed, this field will
+  // be populated with the error.
+  string kubernetes_error = 4;
```

Comment:
Why don't we let the request fail instead and return an error that way?

Reply:
It can happen with valid input, independent of kubernetes - e.g. namespace not existing.